src/System.ComponentModel.Annotations/tests/Helpers.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations.Tests
+{
+    public static class Helpers
```

Comment:
Would it be straightforward to do this as a base class for all of the other relevant attribute test classes?  That way each test class is testing the relevant attribute, rather than having some of the tests for an attribute live in this separate helper class via an entry in TestData.